docs/index.rst

```diff
@@ -0,0 +1,57 @@
+.. F5 OpenStack LBaaSv1 documentation master file, created by
+   sphinx-quickstart on Mon Feb  8 14:03:09 2016.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+F5 OpenStack LBaaSv1
+================================================
+
+Contents:
+
+.. toctree::
+   :maxdepth: 2
```

Comment:
I changed it to `:maxdepth: 1` for now. I can change it later if/when it becomes necessary.